src/YQPkgStatusFilterView.cc

```diff
@@ -63,8 +63,9 @@ void YQPkgStatusFilterView::connectWidgets()
 {
     // Use Qt introspection to recursively find all the QCheckBox children of
     // this filter view and bulk-connect them all to the same slot
+    const auto& children = findChildren<QCheckBox *>();
 
-    foreach ( QCheckBox * checkBox, findChildren<QCheckBox *>() )
+    for ( QCheckBox * checkBox : children )
```

Comment:
I deliberately chose not to do this. Why spread this over several lines?

I am pretty sure that the compiler will optimize this subexpression out of the loop anyway; there will certainly be no more than one function call.

And `auto` variables make things worse for anybody reading the code; it basically signals something along the lines of "I have no idea what type this thng returns". This can be totally avoided by leaving the function call in the `for` loop.